src/libs/config.js

```diff
@@ -12,4 +12,4 @@ export const getConfig = () => {
  * Flags for hidden features
  */
 export const isCromwellAppVisible = () => getConfig().isCromwellAppVisible
-export const isDataBrowserFrontPage = () => false
+export const isDataBrowserFrontPage = () => true
```

Comment:
Why do we need the `isDataBrowserFrontPage` function if it always returns a constant?